### ISSUE TITLE
[Feature] add ollama-llava image suport

### ISSUE THREAD

**Issue by seayuns** [Bin-Huang/chatbox#1305]
**Problem Description**
 ollama-llava  was supported image deal , but can not put image to chatbox

**Proposed Solution**
add some spec model config which support image or opend some limited

**Comment by Anton0C** [comment]
Got next error:
```
The current model Ollama (llava:latest) does not support sending images. Recommended model: Chatbox AI 4.
```

But from console all works perfect:
```
♪ ollama run llava "describe this image, put all details in grouped list: /Users/user/Downloads/5.jpeg"
Added image '/Users/user/Downloads/5.jpeg'
 - The image shows a person lying on the ground, surrounded by a variety of items.

- The individual is wearing a red beanie and glasses, along with a black jacket.

- They are holding a camera in their hands.

- The items surrounding them include:
    - A mountain bike with a vibrant design featuri
```

**Comment by nne998** [comment]
Fixed in v1.4.0 😆
